lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -94,6 +94,13 @@ ipv4_arp_timeout:
 ipv4_arp_timeout_non_vlan_interfaces:
   default_only: ~
 
+ipv4_forwarding:
+  kind: boolean
+  default_value: false
+  cli_nexus:
+    config_get_token_append: '/^ip forward$/'
+    config_set_append: "%s ip forward"
```

Comment:
now that feature/grpc is merged, this will need to be:

```yaml
   nexus:
    get_value: '/^ip forward$/'
    set_value: "<state> ip forward"
```